objects/object.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<body>
+	<script>
+		let user = {
+			name: "John",
+			surname: "Smith"
+		}
+		console.log(user);
+		user.name = "Pete";
+		console.log(user);
+		delete user.name;
+		console.log(user);
+		let salaries = {
+			John: 100,
+			Ann: 160,
+			Pete: 130
+		}
+		if (Object.keys(salaries).length != 0) {
```

Comment:
what you have done should be based on loop, not just adding object values, if there is multiple values in a object always use loop. This is not correct